container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -2023,7 +2023,6 @@ configuration:
       type: CACertificate
       value_type: private_key
     description: PEM-encoded CA key.
-    required: true
```

Comment:
Why is the `required` attribute removed? It is kind of redundant for secrets with generators, but there should be no reason to remove the setting.

Reply:
This is just cleanup. Agreed, random cleanup but has to happen anyway. Want me to move it to a separate commit?

Reply:
I don't understand your comment. How is this "cleanup"? The setting is required, so should keep the attribute, even though it doesn't do anything because the generator will always make sure this setting exist.  All other certs have the `required: true` attribute, so I don't know why this "has to happen anyway".

I don't actually care that much about having the attribute itself, but I do want to understand why it has to be removed here, as that would indicate an issue elsewhere.

Reply:
@jandubois I'll PR a cleanup for the role manifest.
the reason to remove it is "it doesn't do anything".

Reply:
ok I removed the removal

Reply:
> @viovanov: I'll PR a cleanup for the role manifest.

Can you please hold back on that until the role re-arrangement has been merged to avoid merge conflicts...